examples/advanced/llm_hf/README.md

```diff
@@ -60,9 +79,10 @@ Note that the `trainer.train()` call is replaced by a `for` loop, and the three
 
 This setting (1 epoch per round) is for simplicity of this example. In practice, we can set the number of rounds and local epoch per round according to the needs: e.g. 2 rounds with 2 epochs per round will result in 4 training epochs in total.
 
-At the beginning of each round, we intentionally load a fixed model weights saved at the beginning, over-writing the previous round's saved model weights, then call `trainer.train(resume_from_checkpoint=True)` with `trainer.args.num_train_epochs` incremented by 1 so that previous logging results are not overwritten. 
+One important requirement from federated learning is that the model weights need to be set to the "global model" at the beginning of each round.
+Therefore, the iterative training needs to explicitly reflect this behavior - at the beginning of each round, we intentionally load a fixed model weights, over-writing the previous round's saved model weights, then call `trainer.train(resume_from_checkpoint=True)` with `trainer.args.num_train_epochs` incremented by 1 so that previous logging results are not overwritten. 
```

Comment:
A key requirement in federated learning is that model weights must be reset to the "global model" at the beginning of each round. To implement this, we modify the training process to explicitly load the fixed model weights at the start of each round, overwriting the previous round's model. Then, we call trainer.train(resume_from_checkpoint=True) with trainer.args.num_train_epochs incremented by 1 to ensure that logging results from previous rounds are preserved.